extensions/azdata/src/azdata.ts

```diff
@@ -136,7 +136,7 @@ export class AzdataTool implements IAzdataTool {
 					// to get the correct stderr out. The actual value we get is something like
 					// ERROR: { stderr: '...' }
 					// so we also need to trim off the start that isn't a valid JSON blob
-					err.stderr = JSON.parse(err.stderr.substring(err.stderr.indexOf('{'))).stderr;
+					err.stderr = JSON.parse(err.stderr.substring(err.stderr.indexOf('{'), err.stderr.indexOf('}') + 1)).stderr;
```

Comment:
I'm confused about this, what does the original JSON value look like? If its 
```
{ stderr: '...' }
```

then thats valid JSON?

Reply:
Nope, it's like :

```
ERROR: { stderr: '...' }
```

(apparently there's some other stuff that comes at the end too which is what Arvind was fixing)

Reply:
This is just a string that has a JSON blob embedded in it.